### PR TITLE
bug(RHINENG-3777): Fix tasks popover formatting

### PR DESCRIPTION
**Pull request by johnsonm325**
Go to tasks app. In the page header, you'll see the Question mark circle icon. Click it to see the popover and notice the formatting.

Check Image Builder for same type of popover with different filtering. This PR matches the formatting there as requested by QE.

Check Jira card for additional details.

### CODE REVIEW

**Review by mkholjuraev — COMMENTED** [review]
Question: I see that a few custom CSS are being introduced, I am wondering if we achieve the requirement by Patternfly components? I think there are different attributes that maybe useful.

Commit message needs an update from **bug(RHINENG-3777): Fix tasks popover formatting** to **fix(RHINENG-3777): Fix tasks popover formatting**

**Comment by johnsonm325** [comment]
@mkholjuraev I'm not sure how to achieve this with the patternfly components. Maybe you can point me in the right direction?

**Comment by AsToNlele** [comment]
I believe he meant patternfly css classes.
Instead of using `padding-right: 0px;` style to use `pf-u-pr-0` class, `display: inline` to `pf-u-display-inline` class

**Comment by mkholjuraev** [comment]
> @mkholjuraev I'm not sure how to achieve this with the patternfly components. Maybe you can point me in the right direction?

Ah, sorry I missed this ^ and seems @AsToNlele has already made good suggestions.

**Comment by johnsonm325** [comment]
> I believe he meant patternfly css classes. Instead of using `padding-right: 0px;` style to use `pf-u-pr-0` class, `display: inline` to `pf-u-display-inline` class

@AsToNlele where can I find these definitions?

**Comment by johnsonm325** [comment]
@mkholjuraev ready for a re-review.

File: src/constants.js [hunk 1]
```diff
@@ -8,13 +8,18 @@ import {
   ExternalLinkAltIcon,
   OutlinedQuestionCircleIcon,
 } from '@patternfly/react-icons';
+import {
+  Text,
+  TextContent,
+} from '@patternfly/react-core/dist/js/components/Text';
```
Comment (bastilian) [inline src/constants.js:14]: Why not just use `'@patternfly/react-core'`?

File: src/constants.js [hunk 2]
```diff
@@ -8,13 +8,18 @@ import {
...
 import TasksPopover from './PresentationalComponents/TasksPopover/TasksPopover';
 import CompletedTaskDetailsKebab from './SmartComponents/CompletedTaskDetailsKebab/CompletedTaskDetailsKebab';
 import {
   Skeleton,
   SkeletonSize,
 } from '@redhat-cloud-services/frontend-components/Skeleton';
 import RunTaskButton from './PresentationalComponents/RunTaskButton/RunTaskButton';
+import { Button } from '@patternfly/react-core/dist/js/components/Button';
```
Comment (bastilian) [inline src/constants.js:22]: Then this `Button` could also be in the `import` above.

File: src/constants.js (hunk 1 above)
Comment (johnsonm325) [reply src/constants.js:14]: I thought we were optimizing our imports by importing form the specific file.

**Comment by johnsonm325** [comment]
@bastilian This can be reviewed again.

**Review by mkholjuraev — APPROVED** [review]
LGTM! @johnsonm325 thank you. Tested locally, did not spot issues

**Pull request merged**